config/initializers/rateit_app.rb

```diff
@@ -1,2 +1,3 @@
 # frozen_string_literal: true
 # Rateitapp.plugin_manager.add('name_of_your_plugin')
+Rateitapp.oauth_domain = 'accounts.publicradio.org'
```

Comment:
I'd add the initializer to the spec dummy. And set it to something like oauth-provider.example.com.